openjdk/mmtkMutator.cpp

```diff
@@ -34,3 +38,7 @@ HeapWord* MMTkMutatorContext::alloc(size_t bytes, Allocator allocator) {
 void MMTkMutatorContext::flush() {
   ::flush_mutator((MMTk_Mutator) this);
 }
+
+void MMTkMutatorContext::destroy() {
+  ::destroy_mutator((MMTk_Mutator) this);
+}
```

Comment:
This file doesn't end with a newline character.  Although C++11 doesn't forbid it, it may cause problems in unexpected ways, especially when we process files using scripts.